### DISCUSSION TITLE
Arduino Appears to Function, Yet no Display Output

### DISCUSSION THREAD

**Discussion by jamesondiskin** [sanni/cartreader#711] [Hardware]
Hey everyone! I've been using the [Dubesinhower guide](https://www.youtube.com/watch?v=9ynlJS7W4RU) to assemble my Cartridge Reader, and yet, when I get to the step to test the Cartridge Reader (around 22 minutes into the video), and yet when I go to turn on the Cartridge Reader, I get no display output. I believe I've done everything correctly and I don't see any shorts in my soldering anywhere.

If I need to upload pictures or anything, please let me know as I would really like to get this project back in progress!

I have changed the display to use the Serial Monitor instead and get this as a result.
![image](https://user-images.githubusercontent.com/102488831/219816811-7b1310f5-162a-44b2-ab95-c46a7d56a772.png)

Thank you!

**Comment by sanni** [comment]
Check those pins for bad solder joints or solder bridges, maybe even from the factory in case of the Mega Pro.

![qffffaffffff](https://user-images.githubusercontent.com/19169494/213994470-7e05a786-7bce-4679-9cfd-23cb053175c3.jpeg)

![rqwrqqfqrrqrq](https://user-images.githubusercontent.com/19169494/213994476-73da72a9-85a7-4aac-95f7-edccae2f59f5.jpeg)

![fqwraqfqffqfqfqfq](https://user-images.githubusercontent.com/19169494/213995541-164888e7-0a33-4cd4-9c6a-a37a40cffd48.jpg)

**Reply by jamesondiskin** [reply]
All the solder joints seem fine (no cracks, pads fully covered, no bridges) and I've tried to exFAT SD cards (one was a microSD card in an adapter). The database files match that of the current version (V12.3), and the SD cards work. I did bridge the jumper on the screen as described by the guide, if that makes a difference. I also tried a second MKS Mini to no avail.
